packages/react-dev-utils/FlowTypecheckPlugin.js

```diff
@@ -110,22 +110,19 @@ function getFlowVersion(options) {
 function initializeFlow(projectPath, flowconfig, otherFlowTypedDefs) {
   const flowconfigPath = path.join(projectPath, '.flowconfig');
   const gitignorePath = path.join(projectPath, '.gitignore');
-  return getFlowVersion().then(localVersion => Promise.all([
+  return getFlowVersion().then(localVersion => 
     getFlowVersion({global: true}).catch(() => localVersion)
     .then(globalVersion =>
       globalVersion !== localVersion ?
         Promise.reject(new Error(
-          'Your global flow version does not match react-script\'s flow version.\n\n' +
-          'Having two different versions is likely to restart your flow server ' +
-          'regularly while you go back and forth between your App and your IDE ' +
-          '(that is likely to rely on your global flow installation). ' +
-          'This will slow down significantly your development experience. ' +
-          'In order to avoid this, ensure you have flow ' + localVersion + ' globally ' +
-          '(your current global version is ' + globalVersion + ').\n\n' +
-          'Run `npm install -g flow-bin@' + localVersion + '` to fix this.'
+          'Flow integration was disabled because the global Flow version does not match.\n' +
+          'You may either remove the global Flow installation or install a compatible version:\n' +
+          '  npm install -g flow-bin@' + localVersion
```

Comment:
Like this?

<img width="922" alt="2__yarn_start__node_" src="https://cloud.githubusercontent.com/assets/246520/21112370/d0ed77ee-c0a5-11e6-9dae-8e2a521e4c05.png">